gcn/notices/core/Statistics.schema.json

```diff
@@ -9,6 +9,14 @@
       "type": "number",
       "description": "False alarm rate: the rate of occurrence of non-astrophysical events that are of the same intensity or significance as the current event [Hz]"
     },
+    "net_count_rate": {
+      "type": "number",
+      "description": "Net count rate of the transient above the background [counts/s]"
```

Comment:
The description should specify that this is over the rate_duration and rate_energy_range.